File: ts/models/CartItemRequest.ts
import {TicketModel} from "./TicketModel";

export class CartItemRequest {
    public eventId:number;
    public quantity:number;
    public tickets:Array<TicketModel>;

    public isOrderByQuantity():boolean {
        return this.tickets==null || this.tickets.length==0;
    }
}

File: src/main/java/com/columbustheater/viewmodels/CartItemRequest.java
package com.columbustheater.viewmodels;

import com.columbustheater.models.Ticket;

public class CartItemRequest {
    private int eventId;

    private int quantity;

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    private Ticket[] tickets;

    public Ticket[] getTickets() {
        return tickets;
    }

    public void setTickets(Ticket[] tickets) {
        this.tickets = tickets;
    }

    public boolean isOrderByQuantity() {
        return tickets==null || tickets.length==0;
    }

    public int getEventId() {
        return eventId;
    }

    public void setEventId(int eventId) {
        this.eventId = eventId;
    }
}
